cmd/lifecycle/detector.go

```diff
@@ -53,34 +63,75 @@ func (d *detectCmd) Privileges() error {
 }
 
 func (d *detectCmd) Exec() error {
-	group, plan, err := d.detect()
+	dr, err := d.detect()
 	if err != nil {
 		return err
 	}
-	return d.writeData(group, plan)
+	return d.writeData(dr)
 }
 
-func (da detectArgs) detect() (lifecycle.BuildpackGroup, lifecycle.BuildPlan, error) {
+func (da detectArgs) mergeOrderWithStackBuildpacks(order lifecycle.BuildpackOrder) (lifecycle.BuildpackOrder, error) {
```

Comment:
For readability, it might be nice to call this function something like `prependOrderWithStackBuildpacks` to make it clear that stack buildpacks will come first.